app/controllers/application_controller.rb

```diff
@@ -254,6 +254,13 @@ def load_users
     @users = User.accessible_by(current_ability).order(name: :asc)
   end
 
+  def set_current_step
+    return unless current_intake.present?
```

Comment:
This is why we need to know current_intake, and why current intake needs to be the intake of the current_client.